Gemfile.lock

```diff
@@ -338,4 +338,4 @@ DEPENDENCIES
   xcpretty-travis-formatter
 
 BUNDLED WITH
-   2.2.33
+   2.2.32
```

Comment:
Damn, sorry I missed this.

The latest Bundler at this time is 2.3.6. Rather than fixing this, I'll re-bundle with that one when doing the 19.2 code freeze.

Reply:
See 2a411ca334